### PR TITLE
Add OVN db cluster support

### PR DESCRIPTION
**Pull request by numansiddique**
Set the env variable OVN_DB_CLUSTER=yes when 'start' to enable
raft clustering.

Signed-off-by: Numan Siddique <nusiddiq@redhat.com>

### CODE REVIEW

**Comment by flavio-fernandes** [comment]
> Set the env variable OVN_DB_CLUSTER=yes when 'start' to enable
> raft clustering.
> 
> Signed-off-by: Numan Siddique [nusiddiq@redhat.com](mailto:nusiddiq@redhat.com)

Very cool! Will try it out and merge it if I hit not issues. Knowing you, I'm pretty sure I will not. ;)

File: ovn_cluster.sh [hunk 1]
```diff
@@ -231,6 +261,34 @@ function wait-containers() {
...
+    ${RUNC_CMD} exec ${CENTRAL_NAME}-1 ${OVNCTL_PATH} --db-nb-addr=170.168.0.2 --db-nb-create-insecure-remote=yes  \
+--db-sb-addr=170.168.0.2 --db-sb-create-insecure-remote=yes --db-nb-cluster-local-addr=170.168.0.2 \
+--db-sb-cluster-local-addr=170.168.0.2 start_ovsdb
+
+    ${RUNC_CMD} exec ${CENTRAL_NAME}-2 ${OVNCTL_PATH} --db-nb-addr=170.168.0.3 --db-nb-create-insecure-remote=yes  \
+--db-sb-addr=170.168.0.3 --db-sb-create-insecure-remote=yes \
+--db-nb-cluster-local-addr=170.168.0.3 --db-nb-cluster-remote-addr=170.168.0.2 \
+--db-sb-cluster-local-addr=170.168.0.3 --db-sb-cluster-remote-addr=170.168.0.2 start_ovsdb
+
+    ${RUNC_CMD} exec ${CENTRAL_NAME}-2 ${OVNCTL_PATH} --db-nb-addr=170.168.0.4 --db-nb-create-insecure-remote=yes  \
```
Comment (dceara) [inline ovn_cluster.sh]: @numansiddique I guess this should be `${CENTRAL_NAME}-3` if I'm reading it correctly, right?

Comment (numansiddique) [reply ovn_cluster.sh]: Oops. Indeed.

Comment (numansiddique) [reply ovn_cluster.sh]: Fixed it.

**Pull request merged**